lib/perl/Genome/Test/Factory/InstrumentData/AlignmentResult.pm

```diff
@@ -9,9 +9,17 @@ use Genome::Test::Factory::SoftwareResult::User;
 
 sub generate_obj {
     my $self = shift;
+    my %params = @_;
 
-    my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash();
-    return Genome::InstrumentData::AlignmentResult::Bwa->__define__(@_, _user_data_for_nested_results => $result_users);
+    my $build = delete($params{'build'});
+    
+    my $result_users;
+    unless ($build) {
```

Comment:
I find the ```unless ($build)``` construct to be touch difficult to interpret vs a more standard ```if ($build)``` (with proper rearrangement below of course).